component/attr_cache/attr_cache.go

```diff
@@ -767,9 +736,13 @@ func (ac *AttrCache) TruncateFile(options internal.TruncateFileOptions) error {
 		defer ac.cacheLock.RUnlock()
 
 		// no need to truncate the name of the file
-		value, found := ac.cacheMap[options.Name]
-		if found && value.valid() && value.exists() {
-			value.setSize(options.Size)
+		value, err := ac.cacheMap.get(options.Name)
+		if err != nil {
+			log.Err("could not find attribute item in cache to truncate file due to the following error: ", err)
+		} else {
```

Comment:
-Fix error message style.
-The `get()` error needs a different variable name to avoid contaminating the return value.
-It would be cleaner to return in the if block, drop the else and deindent.

Reply:
is this in FlushFile() or TruncateFile? I applied these changes to both of those.